packager/util/config.go

```diff
@@ -24,13 +24,17 @@ import (
 )
 
 type Config struct {
-	LocalOnly bool
-	Port      int
-	CertFile  string // This must be the full certificate chain.
-	KeyFile   string // Just for the first cert, obviously.
-	OCSPCache string
+	LocalOnly	bool
+	Port		int
+	CertFile	string // This must be the full certificate chain.
+	KeyFile		string // Just for the first cert, obviously.
+	NewCertFile	string // The new full certificate chain replacing the expired one.
```

Comment:
Can you explain the semantics here? Currently, CertFile is only read by packager. Is NewCertFile read/write? What happens in 180-days with NewNewCertFile? Or is CertFile now read/write, with some process moving NewCertFile to CertFile before it's time for NewNew?

Reply:
NewCertFile will be read/write.  I envision that both CertFile and NewCertFile will both be set when both are valid (overlap) and that once CertFile becomes invalid, NewCertFile will replace it (CertFile = NewCertFile) and NewCertFile will be empty.  I would think that this applies to the disk copies as well (which may need to be some sort of shared disk, if multiple replicas of amppackager are running).

Reply:
OK, SGTM! Thanks for clarifying.